test/mocks/router/mocks.h

```diff
@@ -417,6 +435,8 @@ class MockRouteEntry : public RouteEntry {
   TestVirtualCluster virtual_cluster_;
   TestRetryPolicy retry_policy_;
   testing::NiceMock<MockInternalRedirectPolicy> internal_redirect_policy_;
+  testing::NiceMock<MockPathMatcher> extension_path_matcher_;
```

Comment:
again, please remove the "extension_" prefix.

Reply:
changed